Ports/cmatrix/package.sh

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=cmatrix
 useconfigure=true
-version=git
+version=3112b127babe72d2222059edd2d7eb7fb8bddfb1
 depends=("ncurses")
-workdir=cmatrix-master
+workdir=${port}-${version}
```

Comment:
This is already the default value for `$workdir`, so you can leave this out.

Reply:
Ok, I removed that line.